XenModel/FriendlyNames.resx

```diff
@@ -1277,6 +1286,15 @@
   <data name="Message.name-vcpu_qos_failed" xml:space="preserve">
     <value>VM vCPU QoS Failed</value>
   </data>
+  <data name="Message.name-vdi_cbt_metadata_inconsistent" xml:space="preserve">
+    <value>Changed block tracking disabled due to metadata inconsistent</value>

```

Comment:
I think it should be either "metadata inconsistency" or "the metadata being inconsistent". I prefer the first though.

Reply:
My suggestion is `Changed block tracking disabled due to inconsistent metadata`

Reply:
Cool. I'd prefer @thomasmck 's version. Let's take that.